Src/BootCamp.Chapter1/Program.cs

```diff
@@ -1,9 +1,13 @@
-﻿namespace BootCamp.Chapter1
+﻿using System;
+
+namespace BootCamp.Chapter1
 {
     class Program
     {
         static void Main(string[] args)
         {
+            int[] testArr = new int[] { 1, 1, 1 };
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the array type; it is redundant.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5075368)